### PR TITLE
Updated controllers for User and blogs to be App

### PR DESCRIPTION
**Pull request by ltramos7**
updated the things

to test this, run `bundle exec shotgun` or `bundle exec rspec`

### CODE REVIEW

File: app/controllers/user.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class App < Sinatra::Base
```
Comment (mattconzen) [inline app/controllers/user.rb:1]: Cool, makes sense for this to be a separate controller!

File: app/views/users/new.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1>User Sign up!</h1>
+
+<form class="user-form" action="/users/new" method="post">
+  <input type="text" name="user[first_name]" placeholder="Enter your first name">
+  <input type="text" name="user[last_name]" placeholder="Enter your last name">
+  <br>
+  <input type="submit" value="Sign Up">
```
Comment (mattconzen) [inline app/views/users/new.erb:7]: Should this be "Create" instead of "Sign Up"?

**Review by mattconzen — APPROVED** [review]
Looks good! Left two minor comments, not blocking. Feel free to merge to `main`.

File: app/views/users/new.erb (hunk 2 above)
Comment (ltramos7) [reply app/views/users/new.erb:7]: Yes, you're right! Thank you:)

File: README.md [hunk 3]
```diff
@@ -3,8 +3,8 @@
 # Install
 - run `bundle`
 
-# Run
-- run `bundle exec ruby myapp.rb`
+# Run Server
+- run `bundle exec shotgun`
```
Comment (ltramos7) [inline README.md:7]: @mattconzen does this make sense?

**Pull request merged**